zstd_stream_test.go

```diff
@@ -398,6 +403,27 @@ func TestStreamWriteNoGoPointers(t *testing.T) {
 	})
 }
 
+func TestStreamSetNbWorkers(t *testing.T) {
+	// Build a big string first
+	sb := strings.Builder{}
```

Comment:
you are not testing the `strings.Builder{}` for a one-liner, I would just `s := strings.Repeat("foobaa", 1000*1000)`